qutebrowser/components/misccommands.py

```diff
@@ -235,23 +236,39 @@ def click_element(tab: apitypes.Tab, filter_: str, value: str, *,
         value: The value to filter for.
         target: How to open the clicked element (normal/tab/tab-bg/window).
         force_event: Force generating a fake click event.
+        select_first: Select first matching element if there are multiple.
     """
-    def single_cb(elem: Optional[apitypes.WebElement]) -> None:
-        """Click a single element."""
-        if elem is None:
-            message.error("No element found with id {}!".format(value))
-            return
+    def do_click(elem: apitypes.WebElement) -> None:
         try:
             elem.click(target, force_event=force_event)
         except apitypes.WebElemError as e:
             message.error(str(e))
+
+    def single_cb(elem: Optional[apitypes.WebElement]) -> None:
+        """Click a single element."""
+        if elem is None:
+            message.error(f"No element found matching {filter_}={value}!")
+            return
+
+        do_click(elem)
+
+    def multiple_cb(elems: List[apitypes.WebElement]) -> None:
+        if len(elems) == 0:
```

Comment:
```suggestion
        if not elems:
```

is typically used for lists in Python, since an empty list evaluates to false.